File: src/test/resources/java/MissingImports.java
public class MissingImports {
    private Random random = new Random();
    private SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
    private File file = new File("example.txt");
    private LocalDate localDate = LocalDate.now();
    private ZoneId zoneId = ZoneId.systemDefault();

    public void printRandomNumber() {
        System.out.println(random.nextInt(100));
    }

    public void printCurrentDate() {
        System.out.println(dateFormat.format(new Date()));
    }

    public void printFileExists() {
        System.out.println(file.exists());
    }

    public void printLocalDate() {
        System.out.println(localDate);
    }

    public void printZoneId() {
        System.out.println(zoneId);
    }
}

File: src/test/resources/kt/MissingImports.kt
class MissingImports {
    private val random = Random()
    private val dateFormat = SimpleDateFormat("yyyy-MM-dd")
    private val file = File("example.txt")
    private val localDate = LocalDate.now()
    private val zoneId = ZoneId.systemDefault()

    fun printRandomNumber() {
        println(random.nextInt(100))
    }

    fun printCurrentDate() {
        println(dateFormat.format(Date()))
    }

    fun printFileExists() {
        println(file.exists())
    }

    fun printLocalDate() {
        println(localDate)
    }

    fun printZoneId() {
        println(zoneId)
    }
}
